README.md

```diff
@@ -126,6 +126,50 @@ const PayWithCoinbaseButton: React.FC = () => {
 };
 ```
 
+## React-Native example
+
+### Prerequisites
+
+``` 
+yarn add react-native-url-polyfill/auto
```

Comment:
```suggestion
yarn add react-native-url-polyfill
```

Reply:
@Perronef5 Could you merge or rebase off of master & push a new commit so we could merge this? 🙏

Reply:
Will do 👍🏽

Reply:
Done